packages/flutter_tools/test/general.shard/version_test.dart

```diff
@@ -394,29 +394,44 @@ void main() {
     const String hash = 'abcdef';
     GitTagVersion gitTagVersion;
 
-    // legacy tag release format
-    gitTagVersion = GitTagVersion.parse('v1.2.3-4-g$hash');
-    expect(gitTagVersion.frameworkVersionFor(hash), '1.2.4-pre.4');
+    // legacy tag format (x.y.z-dev.m.n), master channel
```

Comment:
Would a regression test of the doctor also be useful?

Reply:
The diff in this PR for `dev/bots/test.dart` was because changing the output of `frameworkVersionFor` caused the `verifyVersion` integration test to fail. I think between that integration test and this unit test, we're covering any territory a doctor test would assert. What do you think?